recipe/meta.yaml

```diff
@@ -146,9 +155,9 @@ outputs:
       - targets/{{ target_name }}/include                               # [linux]
       - targets/{{ target_name }}/lib/*.so                              # [linux]
       - Library\include                                                 # [win]
-      - Library\lib\cuda.lib                                            # [win]
-      - Library\lib\cudadevrt.lib                                       # [win]
-      - Library\lib\cudart.lib                                          # [win]
+      - Library\lib\{{ target_name }}\cuda.lib                          # [win]
+      - Library\lib\{{ target_name }}\cudadevrt.lib                     # [win]
+      - Library\lib\{{ target_name }}\cudart.lib                        # [win]
```

Comment:
What would this look like if we kept `.lib`s in `Library\lib`?

Reply:
With no `target_name` directory, you cannot install multiple `cuda-cudart-dev_{{ target_platform }}` packages at the same time because the artifacts are not separated and will clobber. However, this is probably not a common use case and is not possible yet for Windows because there is no alternative to x86 for Windows.

Reply:
Without implementing {{ target_name }} for Windows, this PR is just moving the DLL from `cuda-cudart_{{ target_platform }}` to `cuda-cudart`. Which is what [isuruf originally suggested](https://github.com/conda-forge/cuda-cudart-feedstock/issues/13#issuecomment-1805933220).

Reply:
I can actually shrink the diff more by removing the existence tests I added.